pkg/pub_worker/lib/src/analyze.dart

```diff
@@ -201,6 +194,13 @@ Future<void> _analyzePackage(
       logFile.openRead().transform(gzip.encoder),
     );
 
+    // Upload results, if there is any
+    _log.info('api.taskUploadResult("$package", "$version")');
+    final r = await retry(
+      () => api.taskUploadResult(package, version),
+      retryIf: _retryIf,
+    );
+
     // Create BlobIndex
     final index = await builder.buildIndex(r.blobId);
```

Comment:
If you want to, there is an extremely efficient `blobIndex.update` method, if you'd rather update the blobId after the fact.

Reply:
Maybe, in a subsequent change.